src/err/mod.rs

```diff
@@ -206,7 +206,7 @@ impl PyErr {
     ///     assert_eq!(err.to_string(), "TypeError: ");
     ///
     ///     // Case #3: Invalid exception value
-    ///     let err = PyErr::from_value(PyString::new(py, "foo").into());
+    ///     let err = PyErr::from_value(PyString::new_bound(py, "foo").as_gil_ref());
```

Comment:
This will have to wait for the corresponding `PyErr` change, right?

Reply:
Yes, we need to add `PyErr::from_value_bound` I think.

(or add a trait which abstracts over `&PyAny` and `&Bound<PyAny>` just temporarily to ease migration, thought that'd be a new design question and less mechanical.)